service/modules/lin-reg-module/dp_covariance.py

```diff
@@ -0,0 +1,407 @@
+import pandas as pd
+import numpy as np
+import math
+

```

Comment:
Reviewing this file will require some prep on my side. I will get to it tomorrow